### ISSUE TITLE
Modeling a zero-earnings refund in the EITC

### ISSUE THREAD

**Issue by evtedeschi3** [open-source-economics/Tax-Calculator#2092]
Arindrajit Dube has an interesting and seemingly-simple proposal in The Hill to modify the EITC here: https://thehill.com/opinion/finance/412794-an-anti-poverty-tool-with-bipartisan-support-can-be-even-better#.W9EcPM9yqE9.twitter

He calls it the "Earned and Basic Income Tax Credit" (EBITC), but it's really just a straight forward modification of the EITC: instead of getting $0 in credit at $0 of income, with the EITC phased-in from there, filers would get 50% of the maximum EITC benefit at $0 of income, with the remaining half phased-in from there at half of current phase-in rate (so filers still reach the maximum EITC benefit at the same levels of earnings as under current law).

For example, in 2018, a single childless filer would get $260 at $0 of income under the EBITC, and then the EBITC would phase in from there at 3.825% (half of the current law 7.65%) until getting to the current law maximum of $520.

I was trying to think of how to model this in tc. One obvious approach would be to augment the filer with a Personal Refundable Credit that then phases out immediately at half the EITC phase-in rate. The problem though is that the PRC can differ by filing status, but not by number of children. Further augmenting with a New Child Tax Credit runs into the issue that EITC phase-in/out rates are different for 1, 2, or 3+ children.

Any ideas? If not, the simplest modification to tc might be a parameter setting the $0 EITC benefit by eligibility category (and I guess another with the added $0 benefit for married filers). The default for these would be $0 but could be raised as the user sees appropriate (and then the user would have to manually modify the phase-in rates too).

**Comment by martinholmer** [comment]
@evtedeschi3 said in Tax-Calculator issue #2092:
> He calls it the "Earned and Basic Income Tax Credit" (EBITC), but it's really just a straight forward modification of the EITC: instead of getting $0 in credit at $0 of income, with the EITC phased-in from there, filers would get 50% of the maximum EITC benefit at $0 of income, with the remaining half phased-in from there at half of current phase-in rate (so filers still reach the maximum EITC benefit at the same levels of earnings as under current law).

And then after pointing out that this kind of reform cannot be modeled using the existing personal refundable or new child credit parameters, he continued:
> the simplest modification to `tc` might be a parameter setting the [zero-earnings] EITC benefit by eligibility category (and I guess another with the added [zero-earnings] benefit for married filers). The default for these would be $0 but could be raised as the user sees appropriate (and then the user would have to manually modify the phase-in rates too).

Thanks for your suggested Tax-Calculator enhancement @evtedeschi3.  We'd like to make Tax-Calculator so it could analyze this kind of reform.  I have a question about your suggestion.  But before asking the question, here is the low-level code that computes an EITC for a filing unit:
```
def EITCamount(phasein_rate, earnings, max_amount,
               phaseout_start, agi, phaseout_rate):
    """
    Returns EITC amount given specified parameters.
    English parameter names are used in this function because the
    EITC formula is not available on IRS forms or in IRS instructions;
    the extensive IRS EITC look-up table does not reveal the formula.
    """
    eitc = min(phasein_rate * earnings, max_amount)
    if earnings > phaseout_start or agi > phaseout_start:
        eitcx = max(0., (max_amount - phaseout_rate *
                         max(0., max(earnings, agi) - phaseout_start)))
        eitc = min(eitc, eitcx)
    return eitc
```
And here is how that function is called:
```
            po_start = EITC_ps[EIC]
            if MARS == 2:
                po_start += EITC_ps_MarriedJ[EIC]
            eitc = EITCamount(EITC_rt[EIC], earned, EITC_c[EIC],
                              po_start, c00100, EITC_prt[EIC])
```
It may well be that I'm missing something here, but I don't see the need for the second additional parameter you're suggesting (the one for married filers).  The special married-filers parameter is just for the beginning of the phase-**out** of the EITC benefit, which this kind of reform would not affect.  If the new parameter (call it `frac` for the purposes of this discussion) was a fraction (like the 0.5, or 50%, in the EBITC proposal), couldn't we just modify the first statement in the `EITCamount` function to be like on the following line?
```
    eitc = min((1 - frac) * phasein_rate * earnings + frac * max_amount, max_amount)
```
Does this make any sense?

**Comment by evtedeschi3** [comment]
@martinholmer said:
> It may well be that I'm missing something here, but I don't see the need for the second additional parameter you're suggesting (the one for married filers). The special married-filers parameter is just for the beginning of the phase-out of the EITC benefit, which this kind of reform would not affect. If the new parameter (call it frac for the purposes of this discussion) was a fraction (like the 0.5, or 50%, in the EBITC proposal), couldn't we just modify the first statement in the EITCamount function to be like on the following line?

Ah, right you are. Sorry, I was thinking about the EITC on the fly. Yes, being married should not affect any of these phase **in** parameters modified by the EBITC proposal.

**Comment by martinholmer** [comment]
@evtedeschi3,  what's you're preferred time line for adding the enhancement discussed in issue #2092?

**Comment by martinholmer** [comment]
@evtedeschi3, one more question about the EBITC reform proposal.  You originally said the "basic" benefit was for those with zero **income**, but the EITC starts phasing-in at zero **earnings** (not income).  What exactly is the reform proposal?

**Comment by martinholmer** [comment]
@evtedeschi3 said in issue #2092:
> Within the next week? Unless you find it's actually far more cumbersome than it appears.

I'll see what I can do in the next few days.

**Comment by martinholmer** [comment]
@evtedeschi3,  pull request #2094, which is included in the `taxcalc` 0.22.2 conda packages, provides the enhancement you requested in issue #2092.
